src/rtp/srtp.rs

```diff
@@ -147,13 +146,16 @@ impl SrtpContext {
             #[cfg(feature = "_internal_test_exports")]
             Derived::PassThrough => input.to_vec(),
             Derived::Aes128CmSha1_80 {
-                hmac, salt, enc, ..
+                hmac_key,
```

Comment:
How about calling it just `key`, so we can avoid all these destructures to format vertically?

Reply:
Okay!